AdyenCard/Components/GiftCardComponent/GiftCardDetails.swift

```diff
@@ -0,0 +1,50 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import Foundation
+
+/// Contains the details provided by the card component.
+public struct GiftCardDetails: PaymentMethodDetails {
+
+    /// The payment method type.
+    public let type: String
+
+    /// The encrypted card number.
+    public let cardNumber: String
+
+    /// The encrypted security code.
+    public let securityCode: String
+
+    /// The gift card brand
+    public let brand: String
+
+    /// :nodoc:
+    public var holderName: String? = "BALANCE EUR 10000"
```

Comment:
Is this default holder name really necessary?

Reply:
needed for testing

Reply:
I don't see it is used...